### ISSUE TITLE
[Bug] ComicControl silently fails to create thumbnails for WebP images

### ISSUE THREAD

**Issue by pennro-se** [erintheunready/ComicControl#33]
ComicControl v4.2.9

## Description of Problem
When uploading a `.webp` image, ComicControl will successfully upload the image to the site, however it will not generate a thumbnail nor give an error.

## Steps to Reproduce
1. On a working install of ComicControl, go to `/comiccontrol/image-library`
2. Attempt to upload a `.webp` image (I would attach one but Github doesn't support them, ironic)
3. Upload will complete and the image will be successful, but the listing for it on the Image Library page will show a broken file

**Comment by pennro-se** [comment]
The cause of this issue appears to be a lack of case-handling for `.webp` images in [`/comiccontrol/ajax/img-uploader.php`](https://github.com/erintheunready/ComicControl/blob/935724ce894d8c47ba818527a34964906bc166db/comiccontrol/ajax/img-uploader.php#L80). Implementing [imagewebp()](https://www.php.net/manual/en/function.imagewebp.php) there would likely solve the issue.

**Comment by pennro-se** [comment]
adding the follow case seems to cause it to function correctly:
```php
case 'webp': imagewebp($slate, $uploadFilename, 100); break;
```

**Comment by mrencyc** [comment]
Yes I added that to my fork of comiccontrol and can confirm it does the job.
